pkg/translator/azurelogs/metadata.yaml

```
@@ -0,0 +1,3 @@
+status:
+  codeowners:
+    active: [open-telemetry/collector-approvers, atoulme, cparkins]
```

```suggestion
    active: [atoulme, cparkins]
```
The `open-telemetry/collector-approvers` group is added automatically to the code owners file, it doesn't need to be included here.